cosmos_service/src/db/db.py

```diff
@@ -15,7 +15,7 @@
 Base.metadata.create_all(engine)
 SessionLocal = sessionmaker(bind=engine)
 
-def get_job_details(job_id: str) -> CosmosSessionJob:
+def get_job_details(job_id: str, complete_only=True) -> CosmosSessionJob:
```

Comment:
It looks like this is a short-circuit to quickly get the text layer out + available? Can you document it here?

Reply:
Yeah, this lets you get the text layer info out before cosmos processing completes. I'll update the documentation.